test/common/http/access_log/access_log_formatter_test.cc

```diff
@@ -140,6 +141,23 @@ TEST(AccessLogFormatterTest, requestInfoFormatter) {
     EXPECT_CALL(requestInfo, upstreamHost()).WillOnce(Return(nullptr));
     EXPECT_EQ("-", upstream_format.format(header, header, requestInfo));
   }
+
+  {
+    RequestInfoFormatter upstream_format("UPSTREAM_CLUSTER");
+    std::shared_ptr<Upstream::MockHostDescription> host(new Upstream::MockHostDescription());
+    Upstream::MockClusterInfo cluster;
+    EXPECT_CALL(requestInfo, upstreamHost()).WillRepeatedly(Return(host));
```

Comment:
Instead of adding this all here, can you help make the default mocks better by adding host to MockRequestInfo (and add an ON_CALL default), and add cluster to MockHostDescrption (and add an ON_CALL default), and then this tests becomes simpler and other folks don't need boiler plate in the future.